package.json

```diff
@@ -24,51 +24,50 @@
     "dev:test": "mocha ./gulpfile.js/**/*.test.js"
   },
   "dependencies": {
-    "ansi-colors": "^1.0.1",
-    "babel-core": "^6.25.0",
+    "ansi-colors": "^3.2.4",
+    "babel-core": "^6.26.3",
     "babel-loader": "^7.1.1",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-env": "^1.7.0",
```

Comment:
are all the babel plugins purposely not updated to their latest versions? ie. `7.x.x` I could be wrong about this but I think all the latest stuff is in the namespaced packages like `@babel/core` and `@babel/preset-env`

Reply:
Wonder if `npm audit` and `yarn upgrade --latest` didn't pick those up because of the namespace change. Thanks for calling it out

Primary aim of this PR is to clear out security warnings, secondary goal is to clear out other warnings. The first in several steps towards truly bringing things up to date. Will add Babel updates to the list of next steps

Reply:
Updated the PR description to be more clear about this